### DISCUSSION TITLE
non-docker rebuild recommendation

### DISCUSSION THREAD

**Discussion by lgwapnitsky** [Lissy93/dashy#530] [Hot Tips]
I'm not running in docker, so I decided to use inotify to automate the builds when a config.yml file changes.  Syntax for incrontab is as follows:

`/opt/dashy/public       IN_MODIFY       /usr/bin/yarn --cwd /opt/dashy build`

**Comment by lissy93** [comment]
Sweet- thanks for sharing :)

There's also some changes coming up, which should make rebuilding after modifying config a thing of the past!
